### ISSUE TITLE
Integrate project with CI to run tests on Pull Resquest

### ISSUE THREAD

**Issue by barrault01** [suzuki-0000/SKPhotoBrowser#103]
Everything is in the title

**Comment by krummler** [comment]
I agree, we don't have a setup for this right now but this would be a welcome addition 👍

**Comment by tommyming** [comment]
manually close this issue due to inactivity, if you think this is worth to fix/improve, please feel free to reopen it or submit PR
